### ISSUE TITLE
Custom _app.js file cannot import Global css under src/pages folder

### ISSUE THREAD

**Issue by yulafezmesi** [vercel/next.js#31952]
### What version of Next.js are you using?

12.0.4

### What version of Node.js are you using?

14.18.1

### What browser are you using?

Chrome

### What operating system are you using?

Ubuntu

### How are you deploying your application?

Vercel

### Describe the Bug

By default, the application was generated without the src folder. After moving the pages folder to the src folder, global CSS cannot be imported/loaded.  It doesn't throw any error, just couldn't load.

Folder structure:

root/
├─ node_modules/
├─ public/
│  ├─ favicon.ico
├─ src/
│  ├─ pages/
│  │  ├─ index.js
│  │  ├─ _app.js
│  ├─ components/
│  ├─ styles/
│  │  ├─ global.css
├─ .gitignore
├─ .eslintrc.json
├─ .prettierrc
├─ jsconfig.json
├─ next.config.js
├─ postcss.config.js
├─ tailwind.config.js

```javascript

// _app.js

import { AppWrapper } from '@/context/AppContext';
import 'tailwindcss/tailwind.css';
import '@/styles/global.css';

function MyApp({ Component, pageProps }) {
  return (
    <AppWrapper>
      <Component {...pageProps} />
    </AppWrapper>
  );
}

export default MyApp;

```

### Expected Behavior

Global CSS's will import just like the root folder.

### To Reproduce

Create src folder, move pages folder into src folder, import any global CSS file in the _app.js file. You will see global CSS won't be imported.

**Comment by yulafezmesi** [comment]
Totally my mistake, forgot to edit the purge path in tailwind.config.js file.

```
before: purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}']
after:  purge: ['./src/pages/**/*.{js,ts,jsx,tsx}', './src/components/**/*.{js,ts,jsx,tsx}'],
```

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
